DFC.ServiceTaxonomy.ApiFunction.Tests/DFC.ServiceTaxonomy.ApiFunction.Tests.csproj

```diff
@@ -7,10 +7,21 @@
   </PropertyGroup>
```

Comment:
We should treat warnings as errors...
`    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>`